openvm/scripts/plot_effectiveness.py

```diff
@@ -12,11 +12,12 @@ def load_apc_data(json_path):
         data = json.load(f)
     
     return pd.DataFrame([{
-        'effectiveness': item['total_width_before'] / item['total_width_after'],
-        'instructions': len(item['original_instructions']),
-        'frequency': item['execution_frequency'],
-        'total_width_before': item['total_width_before'],
-        'software_version_cells': item['total_width_before'] * item['execution_frequency']
+        'effectiveness': (wb := sum(item['widths_before'].values())) 
```

Comment:
whoa, seems like dark magic

Reply:
Slight preference for extracting a `item_to_row` function, but I guess this works...

Reply:
python scoping is something...